roles/cisco-nexus-stig-role/tasks/SV-221071r999679_rule.yml

```
@@ -0,0 +1,40 @@
+# DISA STIG:
+#   SRG-NET-000018-RTR-000001
+#   Severity: medium
+#   Rule ID: SV-221071r999679_rule
+#   Identifiers: SV-110961, V-101857, CCI-001368
+#   Title: The Cisco switch must enforce information flow control using ACLs.
+#
+# NOTE: mgmt0 is the only routed interface and currently lacks ACL enforcement.
+# This task creates the ACL (EXTERNAL_ACL) and applies it to mgmt0.
+
+- name: "[SV-221071r999679_rule] Configure EXTERNAL_ACL with required flow rules"
+  cisco.nxos.nxos_acl:
+    config:
+      - name: EXTERNAL_ACL
```

```suggestion
      - name: EXTERNAL_MGMT_ACL
```

Since this content is specifically locking down the ``mgmt0`` interface, we likely want to use a variable name that eludes to external traffic for the mgmt0 interfaces vs "all external traffic"